csharp/src/Apache.Arrow/Ipc/ArrowStreamWriter.cs

```diff
@@ -128,6 +129,15 @@ public void Visit(StructArray array)
                 }
             }
 
+            public void Visit(DictionaryArray array)
+            {
+                //Dictionary is serialized separately in Dictionary serialization.
+                //We are only interested in indexes at this context.
```

Comment:
(nits) comments usually have a space between the slashes and the first character. Also `indexes` => `indices`.

```suggestion
                // Dictionary is serialized separately in Dictionary serialization.
                // We are only interested in indices at this context.
```

Reply:
I fixed it.